Makefile.oss

```diff
@@ -12,21 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# The directory for artifacts for the OSS build of Config Sync.
-OSS_DIR := $(OUTPUT_DIR)/oss
-
-# The directory for manifests for the OSS build of Config Sync.
-OSS_MANIFEST_DIR := $(OUTPUT_DIR)/oss/manifests
-
-$(OSS_MANIFEST_DIR): $(OUTPUT_DIR)
-	@echo "+++ Creating the local output dir for manifests: $(OSS_MANIFEST_DIR)"
-	@mkdir -p \
-		$(OSS_MANIFEST_DIR)
-
-.PHONY: build-oss
-build-oss: $(OSS_MANIFEST_DIR) pull-buildenv
-	MANIFEST_DIR=$(OSS_MANIFEST_DIR) ./scripts/build-oss.sh
```

Comment:
The expectation is now that OSS builds also just use `make build-images`, right?

Reply:
Yeah OSS builds would use:
- `build-images` to just build the docker images
- `build-manifests` to just build the manifests
- `(pull|push|retag)-images` to optionally manipulate docker images

`config-sync-manifest` is a convenience target that behaves similar to `build-oss` which will build manifests and build/push images.

The previous behavior by default used a timestamp in the registry prefix, so you couldn't separately run the targets without overriding the `REGISTRY` variable. This change makes it so that the default behavior allows the user to separately run each target at a given commit. This is because the image tag will default to `git describe`, so it will use the same tag for the same SHA.